AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -8105,7 +8105,7 @@ planet.com##.pl-cookies-cta-contain
 oppo.com##div[class^="cp-cookie-tip"]
 gram.pl###cookie-box-container-inner
 ||privacy-mgmt.com/mms^
-baydoner.com##body > .alert-dismissible[role="alert"]
+sadnescity.it,baydoner.com##body > .alert-dismissible[role="alert"]
```

Comment:
I'd not add it as the site has low visitors. Perhaps we can make a generic rule, but I don't see any other sites for this.
Let's wait for @Alex-302.

Reply:
Why not?) Domain is added to the existing rule. If some porn site, we can ignore it.